Lib9c/BlockPolicySource.cs

```diff
@@ -69,14 +69,15 @@ public IBlockPolicy<NCAction> GetPolicy(int minimumDifficulty)
             BlockChain<NCAction> blockChain
         )
         {
-            if (transaction.Actions.Count == 1 &&
-                transaction.Actions.First().InnerAction is ActivateAccount)
-            {
-                return true;
-            }
-
             try
             {
+                if (transaction.Actions.Count == 1 &&
+                    transaction.Actions.First().InnerAction is ActivateAccount aa)
```

Comment:
Why we need `transaction.Actions.Count == 1` statement?

Reply:
I don't have deep thought, but it can be fair because in the normal cases, ActivateAccount should be only one and first action per account.

Reply:
What happens if someone make a typo when they type the first activation key?

Reply:
In that case, the first activation key will be ignored and user should challenge with other key.